src/olympia/devhub/templates/devhub/agreement.html

```diff
@@ -36,7 +39,7 @@
       {% csrf_token %}
       <button id="accept-agreement" type="submit">
         {% if waffle.switch('post-review') %}
-          {{ _('I have read and accept this Agreement and the Rules and Policies') }}
+          {{ _('Accept') }}
```

Comment:
This unfortunately is less readable for people needing accessibility helpers that read buttons. Do you have a screenshot showing what the actual problem is here? Potentially that can be fixed with styling.

Reply:
hmm, okay I noticed this got decided in #6400 so I think that's what we're going to do.